controls/pcidss_4_ocp4.yml

```diff
@@ -0,0 +1,276 @@
+policy: PCI-DSS
+title: Payment Card Industry - Data Security Standard (PCI-DSS)
+id: pcidss_4_ocp4
+version: '4.0'
+source: https://docs-prv.pcisecuritystandards.org/PCI%20DSS/Standard/PCI-DSS-v4_0.pdf
+levels:
+- id: base
+
+controls:
+  - id: '1.1'
```

Comment:
In the control file for [PCI-DSS 3.2.1](https://github.com/ComplianceAsCode/content/blob/master/controls/pcidss_ocp4.yml), the controls were named `Req-1.1`, instead of just `1.1`.

This reflects on the annotations in the rule, e.g:
```console
$ oc get rules ocp4-configure-network-policies -o yaml
apiVersion: compliance.openshift.io/v1alpha1
[...]
kind: Rule
metadata:
  annotations:
    compliance.openshift.io/image-digest: pb-ocp4s4hps
    compliance.openshift.io/profiles: ocp4-stig,ocp4-nerc-cip,ocp4-moderate,ocp4-pci-dss-3-2,ocp4-high,ocp4-pci-dss,ocp4-stig-v1r1,ocp4-moderate-rev-4,ocp4-cis-1-4,ocp4-high-rev-4,ocp4-cis
    compliance.openshift.io/rule: configure-network-policies
    control.compliance.openshift.io/CIS-OCP: 5.3.1
    control.compliance.openshift.io/NERC-CIP: CIP-003-8 R6;CIP-004-6 R3;CIP-007-3
      R6.1
    control.compliance.openshift.io/NIST-800-53: CM-6;CM-6(1)
->> control.compliance.openshift.io/PCI-DSS: Req-1.1.4;Req-1.2;Req-2.2 <<-
    policies.open-cluster-management.io/controls: 5.3.1,CIP-003-8 R6,CIP-004-6 R3,CIP-007-3
      R6.1,CM-6,CM-6(1),Req-1.1.4,Req-1.2,Req-2.2
    policies.open-cluster-management.io/standards: CIS-OCP,NERC-CIP,NIST-800-53,PCI-DSS
```

Would we like to keep the same syntax?

Reply:
I'm not sure why we used `Req` in the previous implementation, unless it was to denote a different between control numbers in the `policies.open-cluster-management.io/controls` annotation below.